Form-Controls/index.html

```diff
@@ -10,18 +10,41 @@
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1>T-shirt Order Form</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+  <form id="order-form">
+    <!-- Form fields must include a validated name, validated email address, 3 colour options only, 6 sizes only and delivery date -->
+    <!--Customer name-->
+    <div class="form-contents">
+        <label for="Customer name">Your name:</label>
+        <input type="text" id="Customer name" name="Customer name" minlength="2" required placeholder="Please enter your name">
+    </div>
+
+    <!-- Customer email-->
+    <div class="form-contents">
+        <label for="Customer email">Your email address: </label>
+        <input type="email" id="Email address" name="Customer email" required placeholder="Please enter your email address">
+    </div>
+  
+  <!--Color selection-->
+    <div class="form-contents">
+        <label for="T-shirt color">Select your t-shirt colour:</label>
+        <select id="T-shirt color" name="T-shirt color" required>
+        <option value="" disabled selected>Select your colour</option>  
+        <option value="White">White</option>
+        <option value="Black">Black</option>
+        <option value="Grey">Grey</option>
+        </select>
+    </div>
```

Comment:
Please add the delivery date section. Also, please ensure that your netlify link is working.

Reply:
Hi Waldo, I've added the delivery date and the sizes, will test Netlify shortly.